### ISSUE TITLE
Tooltip: Fix axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3420]
**Describe the issue. What is the expected and unexpected behavior?**
- Form elements must have labels (Tooltip position)
- ARIA hidden element must not contain focusable elements (on icon example)

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Run axe extension

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug - more serious axe violations

**What is your product and what release version are you targeting?**

**Comment by jessiehuff** [comment]
Seems to be resolved. Likely closed by #3593
